jupyter_elfinder/elfinder.py

```diff
@@ -1360,34 +1423,26 @@ def __archive(self) -> None:
         os.chdir(cur_cwd)
 
         if os.path.exists(archive_path):
-            self.__content(cur_dir, False)
-            self._response["select"] = [self.__hash(archive_path)]
+            self._response["added"] = [self.__info(archive_path)]
         else:
             self._response["error"] = "Unable to create archive"
 
     def __extract(self) -> None:
         """Uncompress archive."""
-        current = self._request.get("current")
-        target = self._request.get("target")
-        if not current or not target:
+        if not self._options["archivers"]["extract"] or "target" not in self._request:
             self._response["error"] = "Invalid parameters"
             return
-
-        cur_dir = self.__find_dir(self._request["current"], None)
-
-        if not cur_dir:
+        target = self._request["target"]
+        makedir = self._request.get("makedir")
+        cur_file = self.__find(target)
```

Comment:
`target` must be a string here. Do we need to check for that?

Reply:
All fixed in a later commit